src/MentionsInput.js

```diff
@@ -530,7 +530,19 @@ class MentionsInput extends React.Component {
       } else {
         position.left = left
       }
-      position.top = caretPosition.top - highlighter.scrollTop
+
+      // guard for mentions suggestions list clipped by bottom edge of window
+      const tooltipBottom =
+        this.containerRef.getBoundingClientRect().top +
```

Comment:
Since `refs` can be `null` it seems like you need to add an extra check here in case `containerRef` isn't set yet.

Reply:
Added the check to both left and top